added loading to helper test

jc-umana opened the pull request:
Did the makeshift rebase and added ``loading=0.75``

sbillinge commented:
@jc-umana this is still conflicted so something went wrong...

Can you paste what you see in your terminal all the way back to the beginning of the rebase process?

jc-umana commented:
> @jc-umana this is still conflicted so something went wrong...
> 
> Can you paste what you see in your terminal all the way back to the beginning of the rebase process?

Here is everything:
``(regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ git branch
* apthelper
  master
(regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ git checkout master
Switched to branch 'master'
Your branch is ahead of 'origin/master' by 305 commits.
  (use "git push" to publish your local commits)
(regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ git branch -D apthelper
Deleted branch apthelper (was 14073e1).
(regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ git branch
* master
(regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ git checkout -b helpertestedit1
Switched to a new branch 'helpertestedit1'
(regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ git status
On branch helpertestedit1
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   test_helpers.py

no changes added to commit (use "git add" and/or "git commit -a")
(regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ git add .
(regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ git commit -m "added loading to helper test"
[helpertestedit1 6ee2370] added loading to helper test
 1 file changed, 2 insertions(+)
(regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ git push origin helpertestedit1
Username for 'https://github.com': jc-umana
Password for 'https://jc-umana@github.com': 
Enumerating objects: 7, done.
Counting objects: 100% (7/7), done.
Delta compression using up to 2 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 442 bytes | 442.00 KiB/s, done.
Total 4 (delta 3), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
remote: 
remote: Create a pull request for 'helpertestedit1' on GitHub by visiting:
remote:      https://github.com/jc-umana/regolith/pull/new/helpertestedit1
remote: 
To https://github.com/jc-umana/regolith.git
 * [new branch]      helpertestedit1 -> helpertestedit1
(regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ ``

sbillinge commented:
yah, you didn't sync.
```
git checkout master
git pull upstream master
```
only then
```
git checkout -b helpertestedit1
```

On Thu, Jun 18, 2020 at 6:28 AM jc-umana <notifications@github.com> wrote:

> @jc-umana <https://github.com/jc-umana> this is still conflicted so
> something went wrong...
>
> Can you paste what you see in your terminal all the way back to the
> beginning of the rebase process?
>
> Here is everything:
> ``(regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ git branch
>
>    - apthelper
>    master
>    (regolith_env) jc@jc-Inspiron-3195:/dev/regolith/tests$ git checkout
>    master
>    Switched to branch 'master'
>    Your branch is ahead of 'origin/master' by 305 commits.
>    (use "git push" to publish your local commits)
>    (regolith_env) jc@jc-Inspiron-3195:/dev/regolith/tests$ git branch -D
>    apthelper
>    Deleted branch apthelper (was 14073e1
>    <https://github.com/regro/regolith/commit/14073e147cc8534af5c7c01d2c6b4b9224583794>
>    ).
>    (regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ git branch
>    - master
>    (regolith_env) jc@jc-Inspiron-3195:/dev/regolith/tests$ git checkout
>    -b helpertestedit1
>    Switched to a new branch 'helpertestedit1'
>    (regolith_env) jc@jc-Inspiron-3195:/dev/regolith/tests$ git status
>    On branch helpertestedit1
>    Changes not staged for commit:
>    (use "git add ..." to update what will be committed)
>    (use "git restore ..." to discard changes in working directory)
>    modified: test_helpers.py
>
> no changes added to commit (use "git add" and/or "git commit -a")
> (regolith_env) jc@jc-Inspiron-3195:/dev/regolith/tests$ git add .
> (regolith_env) jc@jc-Inspiron-3195:/dev/regolith/tests$ git commit -m
> "added loading to helper test"
> [helpertestedit1 6ee2370
> <https://github.com/regro/regolith/commit/6ee2370096b30f727d9b7c10dda9ed4bcaa0f4f1>]
> added loading to helper test
> 1 file changed, 2 insertions(+)
> (regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ git push origin
> helpertestedit1
> Username for 'https://github.com': jc-umana
> Password for 'https://jc-umana@github.com':
> Enumerating objects: 7, done.
> Counting objects: 100% (7/7), done.
> Delta compression using up to 2 threads
> Compressing objects: 100% (4/4), done.
> Writing objects: 100% (4/4), 442 bytes | 442.00 KiB/s, done.
> Total 4 (delta 3), reused 0 (delta 0), pack-reused 0
> remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
> remote:
> remote: Create a pull request for 'helpertestedit1' on GitHub by visiting:
> remote: https://github.com/jc-umana/regolith/pull/new/helpertestedit1
> remote:
> To https://github.com/jc-umana/regolith.git
>
>    - [new branch] helpertestedit1 -> helpertestedit1
>    (regolith_env) jc@jc-Inspiron-3195:~/dev/regolith/tests$ ``
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/regro/regolith/pull/454#issuecomment-645930435>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABAOWUPM7S7OAKXCC26KH23RXHT5FANCNFSM4OBNYHNA>
> .
>


-- 
Professor Simon Billinge
Columbia University

jc-umana commented:
> yah, you didn't sync. ``` git checkout master git pull upstream master ``` only then ``` git checkout -b helpertestedit1 ```
> […](#)
> On Thu, Jun 18, 2020 at 6:28 AM jc-umana ***@***.***> wrote: @jc-umana <https://github.com/jc-umana> this is still conflicted so something went wrong... Can you paste what you see in your terminal all the way back to the beginning of the rebase process? Here is everything: ``(regolith_env) ***@***.***Inspiron-3195:~/dev/regolith/tests$ git branch - apthelper master (regolith_env) ***@***.***Inspiron-3195:/dev/regolith/tests$ git checkout master Switched to branch 'master' Your branch is ahead of 'origin/master' by 305 commits. (use "git push" to publish your local commits) (regolith_env) ***@***.***Inspiron-3195:/dev/regolith/tests$ git branch -D apthelper Deleted branch apthelper (was [14073e1](https://github.com/regro/regolith/commit/14073e147cc8534af5c7c01d2c6b4b9224583794) <[14073e1](https://github.com/regro/regolith/commit/14073e147cc8534af5c7c01d2c6b4b9224583794)> ). (regolith_env) ***@***.***Inspiron-3195:~/dev/regolith/tests$ git branch - master (regolith_env) ***@***.***Inspiron-3195:/dev/regolith/tests$ git checkout -b helpertestedit1 Switched to a new branch 'helpertestedit1' (regolith_env) ***@***.***Inspiron-3195:/dev/regolith/tests$ git status On branch helpertestedit1 Changes not staged for commit: (use "git add ..." to update what will be committed) (use "git restore ..." to discard changes in working directory) modified: test_helpers.py no changes added to commit (use "git add" and/or "git commit -a") (regolith_env) ***@***.***Inspiron-3195:/dev/regolith/tests$ git add . (regolith_env) ***@***.***Inspiron-3195:/dev/regolith/tests$ git commit -m "added loading to helper test" [helpertestedit1 [6ee2370](https://github.com/regro/regolith/commit/6ee2370096b30f727d9b7c10dda9ed4bcaa0f4f1) <[6ee2370](https://github.com/regro/regolith/commit/6ee2370096b30f727d9b7c10dda9ed4bcaa0f4f1)>] added loading to helper test 1 file changed, 2 insertions(+) (regolith_env) ***@***.***Inspiron-3195:~/dev/regolith/tests$ git push origin helpertestedit1 Username for 'https://github.com': jc-umana Password for ***@***.***': Enumerating objects: 7, done. Counting objects: 100% (7/7), done. Delta compression using up to 2 threads Compressing objects: 100% (4/4), done. Writing objects: 100% (4/4), 442 bytes | 442.00 KiB/s, done. Total 4 (delta 3), reused 0 (delta 0), pack-reused 0 remote: Resolving deltas: 100% (3/3), completed with 3 local objects. remote: remote: Create a pull request for 'helpertestedit1' on GitHub by visiting: remote: https://github.com/jc-umana/regolith/pull/new/helpertestedit1 remote: To https://github.com/jc-umana/regolith.git - [new branch] helpertestedit1 -> helpertestedit1 (regolith_env) ***@***.***Inspiron-3195:~/dev/regolith/tests$ `` — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#454 (comment)](https://github.com/regro/regolith/pull/454#issuecomment-645930435)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABAOWUPM7S7OAKXCC26KH23RXHT5FANCNFSM4OBNYHNA> .
> -- Professor Simon Billinge Columbia University

Ok, third time's a charm! Let's try this again

sbillinge commented:
to be sure we have this, could you explain to me the logic (not the steps).
  Having the logic down is kind of key for success....

On Thu, Jun 18, 2020 at 6:41 AM jc-umana <notifications@github.com> wrote:

> yah, you didn't sync. git checkout master git pull upstream master only
> then git checkout -b helpertestedit1
> … <#m_-7244511097059520810_>
> On Thu, Jun 18, 2020 at 6:28 AM jc-umana *@*.***> wrote: @jc-umana
> <https://github.com/jc-umana> https://github.com/jc-umana this is still
> conflicted so something went wrong... Can you paste what you see in your
> terminal all the way back to the beginning of the rebase process? Here is
> everything: (regolith_env) ***@***.***Inspiron-3195:~/dev/regolith/tests$
> git branch - apthelper master (regolith_env)
> ***@***.***Inspiron-3195:/dev/regolith/tests$ git checkout master Switched
> to branch 'master' Your branch is ahead of 'origin/master' by 305 commits.
> (use "git push" to publish your local commits) (regolith_env)
> ***@***.***Inspiron-3195:/dev/regolith/tests$ git branch -D apthelper
> Deleted branch apthelper (was [14073e1](
> https://github.com/regro/regolith/commit/14073e147cc8534af5c7c01d2c6b4b9224583794)
> <[14073e1](
> https://github.com/regro/regolith/commit/14073e147cc8534af5c7c01d2c6b4b9224583794)>
> ). (regolith_env) ***@***.***Inspiron-3195:~/dev/regolith/tests$ git branch
> - master (regolith_env) ***@***.***Inspiron-3195:/dev/regolith/tests$ git
> checkout -b helpertestedit1 Switched to a new branch 'helpertestedit1'
> (regolith_env) ***@***.***Inspiron-3195:/dev/regolith/tests$ git status On
> branch helpertestedit1 Changes not staged for commit: (use "git add ..." to
> update what will be committed) (use "git restore ..." to discard changes in
> working directory) modified: test_helpers.py no changes added to commit
> (use "git add" and/or "git commit -a") (regolith_env)
> ***@***.***Inspiron-3195:/dev/regolith/tests$ git add . (regolith_env)
> ***@***.***Inspiron-3195:/dev/regolith/tests$ git commit -m "added loading
> to helper test" [helpertestedit1 [6ee2370](
> https://github.com/regro/regolith/commit/6ee2370096b30f727d9b7c10dda9ed4bcaa0f4f1)
> <[6ee2370](
> https://github.com/regro/regolith/commit/6ee2370096b30f727d9b7c10dda9ed4bcaa0f4f1)>]
> added loading to helper test 1 file changed, 2 insertions(+) (regolith_env)
> ***@***.***Inspiron-3195:~/dev/regolith/tests$ git push origin
> helpertestedit1 Username for 'https://github.com': jc-umana Password for
> ***@***.***': Enumerating objects: 7, done. Counting objects: 100% (7/7),
> done. Delta compression using up to 2 threads Compressing objects: 100%
> (4/4), done. Writing objects: 100% (4/4), 442 bytes | 442.00 KiB/s, done.
> Total 4 (delta 3), reused 0 (delta 0), pack-reused 0 remote: Resolving
> deltas: 100% (3/3), completed with 3 local objects. remote: remote: Create
> a pull request for 'helpertestedit1' on GitHub by visiting: remote:
> https://github.com/jc-umana/regolith/pull/new/helpertestedit1 remote: To
> https://github.com/jc-umana/regolith.git - [new branch] helpertestedit1
> -> helpertestedit1 (regolith_env)
> ***@***.***Inspiron-3195:~/dev/regolith/tests$ — You are receiving this
> because you commented. Reply to this email directly, view it on GitHub <#454
> (comment)
> <https://github.com/regro/regolith/pull/454#issuecomment-645930435>>, or
> unsubscribe
> https://github.com/notifications/unsubscribe-auth/ABAOWUPM7S7OAKXCC26KH23RXHT5FANCNFSM4OBNYHNA
> .
> -- Professor Simon Billinge Columbia University
>
> Ok, third time's a charm! Let's try this again
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/regro/regolith/pull/454#issuecomment-645936112>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABAOWUP643E2GGOTYFGZ6CDRXHVLXANCNFSM4OBNYHNA>
> .
>


-- 
Professor Simon Billinge
Columbia University

jc-umana commented:
Okay, so here is the logic behind what we are doing here. Please correct me if I get anything wrong.
To make these changes to this file "mergeable" with the group, I need to make sure that my computer has the same up-to-date files as everyone else in order for only my changes to be reflected, and not an outdated version being put in place of the new file. In order to do this, I need to update the files in accordance to what is available to everyone and is considered up-to-date. Branches are only for working in specific directories to isolate the work that I have done, and I'm only sending an edit from my branch because doing edits outside of my branch would be like me trying to giving you piece of the trunk from my tree when you only asked for twigs from a branch. I was really trying to shoehorn the tree analogy at the end, but it's hard to phrase the logic without saying the steps. I guess it's like you're a lumberjack and you trim branches (edits) from the trunk (branch), but you wouldn't cut down the tree (master) itself while you are in it.

sbillinge commented:
ok, this is pretty good.   I like the lumberjack analogy.

maybe we should move this conversation to a slack thread.  It could be
useful for other people so we could turn it into a kind of tutorial.....

On Thu, Jun 18, 2020 at 7:19 AM jc-umana <notifications@github.com> wrote:

> Okay, so here is the logic behind what we are doing here. Please correct
> me if I get anything wrong.
> To make these changes to this file "mergeable" with the group, I need to
> make sure that my computer has the same up-to-date files as everyone else
> in order for only my changes to be reflected, and not an outdated version
> being put in place of the new file. In order to do this, I need to update
> the files in accordance to what is available to everyone and is considered
> up-to-date. Branches are only for working in specific directories to
> isolate the work that I have done, and I'm only sending an edit from my
> branch because doing edits outside of my branch would be like me trying to
> giving you piece of the trunk from my tree when you only asked for twigs
> from a branch. I was really trying to shoehorn the tree analogy at the end,
> but it's hard to phrase the logic without saying the steps. I guess it's
> like you're a lumberjack and you trim branches (edits) from the trunk
> (branch), but you wouldn't cut down the tree (master) itself while you are
> in it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/regro/regolith/pull/454#issuecomment-645952760>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABAOWUJF5YX5MUQDVEZ7ILTRXHZ4XANCNFSM4OBNYHNA>
> .
>


-- 
Professor Simon Billinge
Columbia University
